zk-kyc/README.md

```diff
@@ -0,0 +1,85 @@
+# zk-KYC Demonstration Application
+
+This demo leverages ID.me api to generate a client authentication token. The token includes a nonce that is associated with the user's connected wallet address, employing principles from the [OpenPubkey: Augmenting OpenID Connect with User held Signing Keys](https://eprint.iacr.org/2023/296) paper. The JWT's integrity is then verified within the guest using ID.me's public RS256 signing [certificates](https://api.id.me/oidc/.well-known/jwks). Subsequently, the guest generates a cryptographic proof of the JWT's integrity and issues a receipt. This receipt, containing the SNARK, an obfuscated identifier, and the user's address, can be validated on the onchain for ERC721 token minting or other transactions, if valid.
+
+For detailed information about the RISC Zero zkVM, refer to our developer [documentation](https://dev.risczero.com/api).
+
+### Getting Started
```

Comment:
I'd love to revise the order of presentation here in order to give devs an opportunity to feel like "wow this was so easy to use" before we overwhelm them with all of the details of the system. 

I think the organization of the [chess example](https://github.com/risc0/risc0/tree/main/examples/chess) is a good reference for what I mean.

Reply:
Perhaps this suggestion is naive in light of the complexity of this demo, but I do think that we should adopt a philosophy of making our explanations as simple/accessible as possible and letting more technical audiences opt-in to the more technical descriptions.

Reply:
I do agree - we should have "wow this was easy to use" when it makes sense. However, I do not think it makes sense for this demo. This is a feature rich demo and thus requires a bit of setup. Unfortunately its not something someone can quickly setup without some web dev knowledge. At least not without some more documentation and details.